interpreter/runtime/memory.ml

```diff
@@ -119,6 +120,7 @@ let store_value mem a o v =
     | I64 x -> x
     | F32 x -> Int64.of_int32 (F32.to_bits x)
     | F64 x -> F64.to_bits x
+    | V128 x -> failwith "TODO v128" (* FIXME V128.to_bits x requires store to accept something other than int64 *)
```

Comment:
Not sure I understand this comment.

Reply:
I concluded that the section (inside of `match`) operates on `I64`, though I might have been wrong.